demos/popover/index.html

```diff
@@ -49,6 +50,17 @@
             trigger: '#triggerBottom'
         }).render();
 
+        var popoverBottom = new Y.Popover({
```

Comment:
You already create this variable.
There is three variable create on this index.html but never used, so could you remove these?
On the first Popover are using a var called 'triggerTop' but is not defined, could you check?